### DISCUSSION TITLE
Product bundles

### DISCUSSION THREAD

**Discussion by garryatbrighton** [YetiForceCompany/YetiForceCRM#17281] [How to]
Product in bundled with another product.
Product A - parent product
Product B - bundled component
Product C - bundled component

If I add to IGIN Product A, the quantity in stock of Product B and Product C is not changing. 
What is missing?

**Comment by MaxiHenkel** [comment]
I also noticed bundles are not working as suppose to. Bundled product should be included in stock + - operations.
So If I sell product:  STRAWBERRY SHAKE and bundled: STRAWBERRIES (100g) and MILK (250ml)
After selling product inventory should be reduced as follows:
STRAWBERRY SHAKE - 1
STRAWBERRIES - 100g
MILK - 250ml
This will need similar to Price List approach where adding bundled product to related list of main product, will trigger popup asking for bundle quantity.
Currently bundled products act as a toy rather than functionality ready to go on production

**Reply by s2b-git** [reply]
Stock control important. If this is working as described, we can break Yetiforce to combination of CRM and ERP. Would be marvelous

**Reply by vovpff** [reply]
You can write code which will calculate stock qty for bundles. This isn't difficult.

Really I'm also don't understand why this functionality not done yet by devs
